config/log4j2.properties

```diff
@@ -0,0 +1,82 @@
+status = error
```

Comment:
It's new default logging configuration shipped with ES 5.x. I'll have a look how to reduce logging here.